app/views/layouts/application.html.haml

```diff
@@ -8,7 +8,7 @@
     %link{href: "http://fonts.googleapis.com/css?family=Lato:300,400,700", rel: "stylesheet", type: "text/css"}/
     = csrf_meta_tags
   %body{class: params[:controller]}
-    .container
+    .container{:class => ("profile" if current_page?(user_path))}
```

Comment:
<code>user_path</code> is throwing an error because we're not passing in an <code>id</code>, so I would just change that to <code>user_path(current_user)</code>